mailpoet/readme.txt

```diff
@@ -145,6 +145,20 @@ We welcome experienced translators to translate directly on [our Transifex proje
 
 Have a question for us? Reach us at security@ our domain, or report security issues to our [Bug Bounty program](https://hackerone.com/automattic).
 
+= Use of 3rd Party Services =
+
+To improve user experience, MailPoet may use the following 3rd party libraries if the _Load 3rd-party libraries_ setting is enabled:
+
+* Google Fonts - used in Form Editor which you can use to customize your forms, and in the Email Editor to style emails. This can be individually [disabled by a filter](https://kb.mailpoet.com/article/332-how-to-disable-google-fonts)
```

Comment:
Should we add a period to the final of this sentence and the two others below it to be consistent with the fourth and fifth items in this list?

Reply:
@rodrigoprimo 
The other lists in the readme only have periods on the last item.
`You can stop loading all these libraries by disabling this...` is not part of the list.

Reply:
I missed that. Thanks for the clarification.